traffic_ops/app/db/migrations/2021030300000000_server_capability_update_cascade.sql

```diff
@@ -0,0 +1,28 @@
+/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE public.server_server_capability DROP CONSTRAINT fk_server_capability;
```

Comment:
Do you need the `public` prefix?

Reply:
Yes, we do. The schema needs to be explicit or some people who have been running ATC since pre-1.4 will be unable to run the migration successfully.